### DISCUSSION TITLE
when updating domain names in server got this below error

### DISCUSSION THREAD

**Discussion by vikramdevopsb79** [learndevopsonline/learndevopsonline#1906] [Q&A]
![Screenshot (417)](https://github.com/learndevopsonline/learndevopsonline/assets/169573156/913c1e20-ddcd-41c8-9ef4-30ec92d168b0)

still the server is active while pushing to remote node getting this error

**Comment by chaithanya1812** [comment]
Hi @vikramdevopsb79,
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/3994cef8-d638-4be8-9845-52ca351ed9f5)

**Reply by vikramdevopsb79** [reply]
thanks got it
